### PR TITLE
102 added Docs.size()

### PR DESCRIPTION
**Pull request by ggajos**
for #102

### CODE REVIEW

**Comment by karato** [comment]
Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@carlosmiranda do this review

File: src/test/java/com/nerodesk/om/mock/MkDocsTest.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+import org.apache.commons.io.FileUtils;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+/**
+ * This provides tests for {@link com.nerodesk.om.mock.MkDocs}
```
Comment (carlosmiranda) [inline src/test/java/com/nerodesk/om/mock/MkDocsTest.java]: No need to use the fully qualified name, just `{@link MkDocs}`. Furthermore, the sentence needs a period at the end (Qulice is complaining about it).

**Comment by carlosmiranda** [comment]
@ggajos Just one minor comment above.

**Comment by ggajos** [comment]
@carlosmiranda thanks for feedback, please have a look if everything is fine now

**Comment by carlosmiranda** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge it

**Comment by karato** [comment]
@carlosmiranda Thank you, I have added **17 mins** to your account in payment/transaction "`54137783`", time consumed: 75 hours and 9 mins. you're getting extra minutes here (c=2). +17 to your rating, your total score is [+4147](http://www.netbout.com/b/35023?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**